lib/compute/index.js

```diff
@@ -439,6 +439,126 @@ Compute.prototype.getAddresses = function(options, callback) {
 };
 
 /**
+ * Get a list of autoscalers. For a detailed description of this method's
+ * options, see [API reference](https://cloud.google.com/compute/docs/reference/v1/autoscalers/aggregatedList).
+ *
+ * @resource [Managing Autoscalers]{@link https://cloud.google.com/compute/docs/autoscaler/managing-autoscalers}
+ * @resource [Understanding Autoscaler Decisions]{@link https://cloud.google.com/compute/docs/autoscaler/understanding-autoscaler-decisions}
+ * @resource [Autoscalers: aggregatedList API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/autoscalers/aggregatedList}
+ *
+ * @param {object=} options - Address search options.
+ * @param {boolean} options.autoPaginate - Have pagination handled
+ *     automatically. Default: true.
+ * @param {string} options.filter - Search filter in the format of
+ *     `{name} {comparison} {filterString}`.
+ *     - **`name`**: the name of the field to compare
+ *     - **`comparison`**: the comparison operator, `eq` (equal) or `ne`
+ *       (not equal)
+ *     - **`filterString`**: the string to filter to. For string fields, this
```

Comment:
Are the hyphens supposed to indicate something? Like a new line perhaps..? I could be wrong, but I don't believe we sniff for anything like that.